### ISSUE TITLE
Python kernel crashes when using Chroma's from_texts from langchain

### ISSUE THREAD

**Issue by Kviilen** [langchain-ai/langchain#26859]
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

from langchain_community.vectorstores import Chroma
embed_model_path = '.././AI-ModelScope/bge-small-en-v1___5'
from langchain_huggingface import HuggingFaceEmbeddings
embedding = HuggingFaceEmbeddings(model_name=embed_model_path)
texts = [
    "Test"
]
try:
    smalldb_chinese = Chroma.from_texts(texts, embedding=embedding)
except Exception as r:
    print('%s' %(r))

### Error Message and Stack Trace (if applicable)

Process finished with exit code -1073741819 (0xC0000005)

### Description

When using the from_texts method of Chroma from langchain, the Python kernel crashes without any error messages. The process finishes with exit code -1073741819 (0xC0000005). This issue occurs consistently and makes it impossible to use this method effectively. I am using the latest version of Chroma from langchain and have tried on different environments but still encounter the same problem. Any help or suggestions to resolve this issue would be greatly appreciated.

### System Info

System Information
------------------
> OS:  Windows
> OS Version:  10.0.22631
> Python Version:  3.12.4 | packaged by Anaconda, Inc. | (main, Jun 18 2024, 15:03:56) [MSC v.1929 64 bit (AMD64)]

Package Information
-------------------
> langchain_core: 0.3.5
> langchain: 0.3.0
> langchain_community: 0.3.0
> langsmith: 0.1.125
> langchain_experimental: 0.3.0
> langchain_huggingface: 0.1.0
> langchain_text_splitters: 0.3.0

Optional packages not installed
-------------------------------
> langgraph
> langserve

Other Dependencies
------------------
> aiohttp: 3.9.5
> async-timeout: Installed. No version info available.
> dataclasses-json: 0.6.7
> httpx: 0.27.0
> huggingface-hub: 0.24.5
> jsonpatch: 1.33
> numpy: 1.26.4
> orjson: 3.10.6
> packaging: 23.2
> pydantic: 2.8.2
> pydantic-settings: 2.5.2
> PyYAML: 6.0.1
> requests: 2.32.2
> sentence-transformers: 3.0.1
> SQLAlchemy: 2.0.30
> tenacity: 8.5.0
> tokenizers: 0.19.1
> transformers: 4.44.0
> typing-extensions: 4.11.0

**Comment by air-kyi** [comment]
My kernel crashes with from_documents too.

## Example Code
```
self.embed_model = AzureOpenAIEmbeddings(api_key=self.api_key, 
                                                                            azure_endpoint=self.endpoint, 
                                                                            model="text-embedding-ada-002", 
                                                                            chunk_size=1)

# load documents
loader = DataFrameLoader(self.data, page_content_column=column)
documents = loader.load()

vectorstore = Chroma.from_documents(documents=documents, 
                                            embedding=self.embed_model, 
                                            persist_directory=persist_directory)

vectorstore.persist()
```

Error Message and Stack Trace (if applicable)
Process finished with exit code -1073741819 (0xC0000005)

## Description
When using the from_documents method of Chroma from langchain, the Python kernel crashes without any error messages. The process finishes with exit code 3221225477. I am using the latest version of Chroma from langchain_chroma and have tried:
- Running on ipython
- Running in terminal

## System Info
OS: Windows
Version	10.0.19045 Build 19045
Python 3.11.1

## Package Information
langchain-chroma==0.1.4
├── chromadb [required: >=0.4.0,<0.6.0,!=0.5.5,!=0.5.4, installed: 0.5.18]
├── fastapi [required: >=0.95.2,<1, installed: 0.115.4]
├── langchain-core [required: >=0.1.40,<0.4, installed: 0.3.15]
└── numpy [required: >=1,<2, installed: 1.26.4]
langchain-community==0.3.5
├── aiohttp [required: >=3.8.3,<4.0.0, installed: 3.10.10]
├── dataclasses-json [required: >=0.5.7,<0.7, installed: 0.6.7]
├── httpx-sse [required: >=0.4.0,<0.5.0, installed: 0.4.0]
├── langchain [required: >=0.3.6,<0.4.0, installed: 0.3.7]
├── langchain-core [required: >=0.3.15,<0.4.0, installed: 0.3.15]
├── langsmith [required: >=0.1.125,<0.2.0, installed: 0.1.140]
├── numpy [required: >=1,<2, installed: 1.26.4]
├── pydantic-settings [required: >=2.4.0,<3.0.0, installed: 2.6.1]
├── PyYAML [required: >=5.3, installed: 6.0.2]
├── requests [required: >=2,<3, installed: 2.32.3]
├── SQLAlchemy [required: >=1.4,<2.0.36, installed: 2.0.35]
└── tenacity [required: >=8.1.0,<10,!=8.4.0, installed: 8.5.0]
langchain-openai==0.2.6
├── langchain-core [required: >=0.3.15,<0.4.0, installed: 0.3.15]
├── openai [required: >=1.54.0,<2.0.0, installed: 1.54.2]
└── tiktoken [required: >=0.7,<1, installed: 0.8.0]

**Comment by anonymousCAVEMAN** [comment]
can forward it another issue is cant load it from saved path even with Chroma. Client().get_or_create....... chroma is cursed idk it doesnt work well with large data sets even thought  i am adding with bath size still even tried mini embeddings no good.

**Comment by jmanuoz** [comment]
Adding the persist_directory makes it work for me

**Comment by anonymousCAVEMAN** [comment]
yes it is must to provide persist directory. still it does not provide any working response. i am able to execute with small db's but not with large ones. shifted to fiasss for large one afterall it provides faster search i didnt wanted to perform any RAG task with chroma
